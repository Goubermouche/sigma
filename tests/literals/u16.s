i32 main() {
	u16 min = 0;
	u16 max = 65535;

	printf("%u %u\n", min, max);

	u16 overflow = 65536;
	u16 underflow = -1;

	printf("%u %u\n", overflow, underflow);

	ret 0;
}
