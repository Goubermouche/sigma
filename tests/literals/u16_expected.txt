0 65535
0 65535
